wgpu-types/src/lib.rs

```diff
@@ -702,7 +702,19 @@ impl DownlevelCapabilities {
 }
 
 bitflags::bitflags! {
-    /// Binary flags listing various ways the underlying platform does not conform to the WebGPU standard.
+    /// Binary flags listing features that may or may not be present on downlevel adapters.
+    ///
+    /// Flags that are **not** present for a downlevel adapter or device usually indicates
+    /// non-compliance with the WebGPU specification, but not always.
+    ///
+    /// You can check whether a set of flags is compliant by by comparing to
+    /// [`DownlevelFlags::compliant()`]:
+    ///
+    /// ```
+    /// # let my_flags = DownlevelFlags::all();
+    /// # use wgpu_types::DownlevelFlags;
+    /// assert_eq!(my_flags & DownlevelFlags::compliant(), DownlevelFlags::compliant());
```

Comment:
We should just be routing people to the DownlevelProperties::is_webgpu_compliant check.